spooldo

```diff
@@ -59,23 +71,35 @@ def archive_incoming(incoming_fname, fname, archive_dnames=[]):
         os.unlink(incoming_fname)
 
 def do_spool(incoming_dname, active_dname, archive_dnames=[], cmd_tmpl=None):
-    for incoming_fname in dirwalk(incoming_dname):
-        rel_fname = os.path.relpath(incoming_fname, incoming_dname)
-        fname = rel_fname.replace(os.path.sep, '--')
-        active_fname = os.path.join(active_dname, fname)
-
-        try:
-            os.link(incoming_fname, active_fname)
-        except:
-            warn('unable to link %s to %s' % (incoming_fname, active_fname))
-
-        if cmd_tmpl:
-            rc = run_cmd(CommandTemplate(cmd_tmpl).substitute(fname=active_fname))
-            if rc != 0:
-                os.unlink(active_fname)
-                error('command failed')
+    jobs = dict()
+    try:
+        limit = int(os.getenv("SPOOLDO_JOBS"))
```

Comment:
I would suggest a comment that SPOOLDO_JOBS is optional

Reply:
Will restrict the value to be >=1.

"Too high" will be system dependent, and should not be hard coded.

README introduces SPOOLDO_JOBS with "if SPOOLDO_JOBS is set ..." which shows optional, as does the "try / set default on exception" stanza.